.github/workflows/yocto-build-deploy.yml

```diff
@@ -384,13 +384,14 @@ jobs:
           API_ENV: ${{ env.BALENARC_BALENA_URL }}
           TRANSLATION: "v6"
           DEVICE_SLUG: ${{ steps.balena-lib.outputs.device_slug }}
+          BALENAOS_TOKEN: ${{ secrets.BALENA_API_DEPLOY_KEY }}
```

Comment:
Afaict that's also how we do it in scripts using curls
https://github.com/balena-os/balena-yocto-scripts/blob/acf945c2b6b9a574bcf3927288805474434e44d1/.github/workflows/yocto-build-deploy.yml#L985-L1033C205